test-files/cmd/slow/ConflateCmdStatsGenericRiversTest.sh.stdout

```diff
@@ -1,10 +1,10 @@
-11:08:51.889 INFO   ...ore/io/OsmMapReaderFactory.cpp( 184) Loading map from ...d/slow/ConflateCmdStatsTest/generic-rivers-out.osm...
-11:08:51.891 INFO   .../hoot/core/io/OsmXmlReader.cpp( 911) Read 0 elements from input.        
11:08:51.891 INFO   .../hoot/core/io/OsmXmlReader.cpp( 911) Read 0 elements from input.        
11:08:51.907 INFO   .../hoot/core/io/OsmXmlReader.cpp( 475) 	Reporting missing elements...
-11:08:51.907 INFO   ...ore/io/OsmMapReaderFactory.cpp( 228) Read 459 elements from input in: 00:00.
-11:08:51.907 INFO   ...ore/io/OsmMapReaderFactory.cpp( 184) Loading map from ...d/slow/ConflateCmdStatsTest/generic-rivers-out.osm...
-11:08:51.908 INFO   .../hoot/core/io/OsmXmlReader.cpp( 911) Read 0 elements from input.        
11:08:51.908 INFO   .../hoot/core/io/OsmXmlReader.cpp( 911) Read 0 elements from input.        
11:08:51.923 INFO   .../hoot/core/io/OsmXmlReader.cpp( 475) 	Reporting missing elements...
-11:08:51.923 INFO   ...ore/io/OsmMapReaderFactory.cpp( 228) Read 459 elements from input in: 00:00.
-11:08:51.927 STATUS .../cpp/hoot/core/cmd/DiffCmd.cpp( 145) Map difference calculated in 00:00 total.
+19:01:59.019 INFO   ...ore/io/OsmMapReaderFactory.cpp( 184) Loading map from ...d/slow/ConflateCmdStatsTest/generic-rivers-out.osm...
```

Comment:
I can't remember exactly now why these stats change, especially since this doesn't use DC. But the output feeding the stats looks better now for some reason, so ok.